src/parse.c

```diff
@@ -1780,11 +1792,11 @@ static const mapping_entry_handler dhcp6_overrides_handlers[] = {
     {"networkmanager", YAML_MAPPING_NODE, NULL, nm_backend_settings_handlers}
 
 /* Handlers for physical links */
-#define PHYSICAL_LINK_HANDLERS                                                           \
-    {"match", YAML_MAPPING_NODE, handle_match},                                          \
-    {"set-name", YAML_SCALAR_NODE, handle_netdef_str, NULL, netdef_offset(set_name)},    \
-    {"wakeonlan", YAML_SCALAR_NODE, handle_netdef_bool, NULL, netdef_offset(wake_on_lan)}, \
-    {"wakeonwlan", YAML_SEQUENCE_NODE, handle_wowlan, NULL, netdef_offset(wowlan)},       \
+#define PHYSICAL_LINK_HANDLERS                                                                \
+    {"match", YAML_MAPPING_NODE, handle_match},                                               \
+    {"set-name", YAML_SCALAR_NODE, handle_netdef_str, NULL, netdef_offset(set_name)},         \
+    {"wakeonlan", YAML_SCALAR_NODE, handle_netdef_bool, NULL, netdef_offset(wake_on_lan)},    \
+    {"wakeonwlan", YAML_SEQUENCE_NODE, handle_wowlan, NULL, netdef_offset(wowlan)},           \
```

Comment:
more unrelated whitespace changes

Reply:
I removed them and will prepare another PR for cleaning this up.